.gitignore

```diff
@@ -10,3 +10,5 @@
 
 /docs/_build/
 /GeoLite2-City.mmdb
+.idea/
+*.pyc
```

Comment:
No need for adding these ignores, I prefer to have these in my global gitignore.